advanced-usage/index.md

```diff
@@ -0,0 +1,372 @@
+<button onclick="toggle_advanced()" id="pirate_toggle">
+    hide advanced functionality
+</button>
+</div>
+
+Any pages that are extended by (or only available in) Pirate Mode are shown in
+[dark mode](#settings), and described with grey text.
+
+### Main
+
+When you first open BlueOS, you'll see a window like the following:
+{{ simple_pirate_image(src="main", width=600) }}
+
+#### Indicators and Network Configuration
+
+On the right side of the header you'll find:
+- Notifications
+   - Press broom to clear notifications
+   - Press cog to toggle showing old messages
+
+{{ easy_image(src="notifications", width=350) }}
+
+- Ethernet + static IP management - choose between:
+   - A static IP
+   - A DHCP server
+   - A dynamic IP
+
+{{ easy_image(src="ethernet", width=350) }}
+{{ easy_image(src="ethernet-example", width=250) }}
+
+- Wifi network management
+   - Choose a network to connect to
+   - Forget, connect to, or a force a new password for a saved network
+{{ easy_image(src="wifi", width=350) }}
+{{ easy_image(src="wifi-example", width=250) }}
+
+- System status
+   - Heartbeat icon pulses with vehicle heartbeat, and goes red if heartbeat is lost
+   - On click shows onboard computer temperature, voltage, and current usage
+   - Additional warning icons appear if onboard computer is too hot or under-powered
+{{ easy_image(src="system-status", width=200) }}
+
+#### Sidebar
+
+The burger menu at the top left of the header opens up the side-bar, for
+conveniently accessing available pages, tools, and services.
+
+{{ simple_pirate_image(src="sidebar", width=200) }}
+
+##### Settings
+
+{{ simple_pirate_image(src="settings", width=300) }}
+
+- Reset BlueOS settings
+   - Remove existing camera/endpoint/bridges configuration
+- Pirate mode toggle
+   - Access or hide advanced functionality
+- Dark mode toggle
+   - Change between light and dark viewing modes
+
+##### Power
+
+{{ simple_pirate_image(src="power", width=300) }}
+
+- Power off
+   - Shut down onboard computer
+   - Recommended before turning off vehicle power
+- Reboot
+   - Reboot onboard computer
+{% simple() %}
+- Soft restart
+   - Do a software-restart of BlueOS (without restarting the computer)
+   - Generally sufficient for most 'reboot' requirements
+{% end %}
+{% pirate() %}
+- Restart core container
+   - Restart the core BlueOS docker container
+   - Generally sufficient for most 'reboot' requirements
+{% end %}
+
+##### Feedback
+
+{{ easy_image(src="feedback", width=300) }}
+
+Submit feedback about BlueOS via:
+- Issues on the GitHub repository (allows easily tracking changes, 
+and notification when complete/fixed)
+- Posts on the Blue Robotics forum (allows easy discussion with the community)
+- Messages in the Blue Robotics Slack (for internal use)
+
+## Vehicle
+
+### General
+The "General" page provides basic info about the active autopilot, along with
+options to:
+- Change board (select a connected board, or run an 
```

Comment:
This option is only available in pirate mode

Reply:
#new-in-1.1